js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,15 @@
+function frequencySort(...args) {
+  if (args[0].length === 0)
+    throw "";
```

Comment:
always try to mention whats the error, with the error message like:
`throw new TypeError ("Array is empty.");`